### PR TITLE
Handle more keypresses

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/mafintosh/neat-input/issues/2

Feel free to squash at merge.

@mafintosh If you enable Travis I can update with badge and `.travis.yml`

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Needs some docs still. So maybe hold of with merge for a while.

**Comment by ralphtheninja** [comment]
Ok. Ready for squash!

File: index.js [hunk 1]
```diff
@@ -52,11 +54,28 @@ function neatInput (opts) {
...
       input.emit('alt-' + key.name)
+      if (key.name === 'b') {
+        input.cursor = util.findWordBeginBackward(rawLine, input.cursor)
+      } else if (key.name === 'f') {
+        input.cursor = util.findWordEndForward(rawLine, input.cursor)
+      } else if (key.name === 'backspace') {
+        deleteWordBackward()
+      } else if (key.name === 'd') {
+        deleteWordForward()
+      }
```
Comment (ralphtheninja) [inline index.js:78]: I kept the same `else if` style here since the first ones are really long. Let me know if I should change this.

**Comment by mafintosh** [comment]
1.8.0

@ralphtheninja enabled travis as well

**Pull request merged**